spec/parsers/rubygems_spec.rb

```diff
@@ -52,7 +52,7 @@
         {:name=>"orm_adapter", :requirement=>"~> 0.1", :type=>:development},
         {:name=>"bcrypt", :requirement=>"~> 3.0", :type=>:runtime},
         {:name=>"thread_safe", :requirement=>"~> 0.1", :type=>:runtime},
-        {:name=>"railties", :requirement=>">= 3.2.6, < 5", :type=>:runtime},
```

Comment:
What's that about?

Reply:
No idea, it's failing on master though. Would love advice from someone who has more context here.

Reply:
As far as I can tell, this is due to https://github.com/ruby/ruby/commit/8da8d4b043c37b53a69803c71ff36b478d4776d0, which is NOT in Ruby 2.5.1, but IS in whatever Travis is calling Ruby 2.5.1

Reply:
@jsonperl @katzj I added consistent sorting of requirements